app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable, :omniauthable, :confirmable
```

Comment:
@marderer are all of the devise extensions needed for a user? should it be `trackable` for example?

Reply:
@AleksSenkou i think that trackable may be doesn't need, but other need